gstd/src/config.rs

```diff
@@ -0,0 +1,75 @@
+// This file is part of Gear.
+
+// Copyright (C) 2021-2022 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! Declares modules, attributes, public re-exports.
+//! Gear libs are `#![no_std]`, which makes them lightweight.
+
+//! This module is for configuring `gstd` inside gear programs.
+use crate::errors::{ContractError, Result};
+
+/// `gstd` configuration
+pub struct Config {
```

Comment:
what about also allow specify default: wait up to / wait for
like I just once write set_default_up_to(500)
and each time I call just await it triggers **up_to** 500